ios/ItaliaApp/Info.plist

```diff
@@ -1,139 +1,144 @@
-    <string>The app needs NSLocationWhenInUseUsageDescription permission</string>
-    <key>NSMicrophoneUsageDescription</key>
-    <string>IO needs access to the microphone in case you want to leave a voice note</string>
-    <key>NSMotionUsageDescription</key>
-    <string>The app needs NSMotionUsageDescription permission</string>
-    <key>NSPhotoLibraryAddUsageDescription</key>
-    <string>The app needs NSPhotoLibraryAddUsageDescription permission</string>
-    <key>NSPhotoLibraryUsageDescription</key>
-    <string>You’ll be able to save bonuses, certificates and upload screenshots or payment notices.</string>
-    <key>NSSpeechRecognitionUsageDescription</key>
-    <string>The app needs NSSpeechRecognitionUsageDescription permission</string>
-    <key>UIAppFonts</key>
-    <array>
-      <string>TitilliumWeb-Black.ttf</string>
-      <string>TitilliumWeb-Bold.ttf</string>
-      <string>TitilliumWeb-BoldItalic.ttf</string>
-      <string>TitilliumWeb-ExtraLight.ttf</string>
-      <string>TitilliumWeb-ExtraLightItalic.ttf</string>
-      <string>TitilliumWeb-Italic.ttf</string>
-      <string>TitilliumWeb-Light.ttf</string>
-      <string>TitilliumWeb-LightItalic.ttf</string>
-      <string>TitilliumWeb-Regular.ttf</string>
-      <string>TitilliumWeb-SemiBold.ttf</string>
-      <string>TitilliumWeb-SemiBoldItalic.ttf</string>
-      <string>Ionicons.ttf</string>
-      <string>RobotoMono-Bold.ttf</string>
-      <string>RobotoMono-BoldItalic.ttf</string>
-      <string>RobotoMono-Light.ttf</string>
-      <string>RobotoMono-LightItalic.ttf</string>
-      <string>RobotoMono-Regular.ttf</string>
-      <string>RobotoMono-RegularItalic.ttf</string>
-      <string>io-icon-font.ttf</string>
-    </array>
-    <key>UIBackgroundModes</key>
-    <array>
-      <string>remote-notification</string>
-    </array>
-    <key>UILaunchStoryboardName</key>
-    <string>LaunchScreen</string>
-    <key>UIRequiredDeviceCapabilities</key>
-    <array>
-      <string>armv7</string>
-    </array>
-    <key>UISupportedInterfaceOrientations</key>
-    <array>
-      <string>UIInterfaceOrientationPortrait</string>
-    </array>
-    <key>UIViewControllerBasedStatusBarAppearance</key>
-    <false/>
-    <key>com.apple.developer.nfc.readersession.iso7816.select-identifiers</key>
-    <array>
-      <string>com.apple.developer.nfc.readersession.iso7816.select-identifiers</string>
-      <string>A0000000308000000009816001</string>
-      <string>A00000000039</string>
-      <string>A0000002471001</string>
-      <string>00000000000000</string>
-    </array>
-    <key>method</key>
-    <string>app-store</string>
-    <key>NFCReaderUsageDescription</key>
-    <string>NFC tag to read NDEF messages into the application</string>
-    <key>LSApplicationQueriesSchemes</key>
-    <array>
-      <string>http</string>
-      <string>https</string>
-    </array>
-  </dict>
-</plist>
\ No newline at end of file
+	<string>1</string>
```

Comment:
I can't run the app using `yarn run-ios`

I got an error and I believe this row should not be here
```
❌  error: unable to read property list from file: /Users/matteo/Documents/Mobile/ReactNative/io-app/ios/ItaliaApp/Info.plist: The operation couldn’t be completed. (XCBUtil.PropertyListConversionError error 2.) (in target 'ItaliaApp' from project 'ItaliaApp')
```

Reply:
😮 I believed too much in automatic checks! I expected the CI give me a problem in this case.
Sorry problem due to merge with master using merging tools!

Fixed in [d2fedb6](https://github.com/pagopa/io-app/pull/3439/commits/d2fedb6b5b300424448ba58b3ba5fc9678a810d9)